Dockerfile

```
@@ -124,41 +149,92 @@ RUN --mount=type=bind,rw,source=.git,target=/app/.git \
 # Copy cache directory and set correct permissions
-COPY --from=builder --chown=appuser:nogroup /var/tmp/youtubei.js /var/tmp/youtubei.js
+COPY --from=builder --chown=10001:10001 /var/tmp/youtubei.js /var/tmp/youtubei.js
 
 # Set the working directory
 WORKDIR /app
 
 COPY --from=builder /app/invidious_companion ./
+COPY --chmod=0444 ./dependencies.json ./
 
 ARG HOST PORT THC_VERSION THC_PORT_NAME TINI_VERSION
 EXPOSE "${PORT}/tcp"
```

Hadolint (2.15.1)

[error] unexpected '"'
expecting '\', A port optionally followed by the protocol, A range of ports optionally followed by the protocol, a new line followed by the next instruction, or at least one space

(DL1000)